src/projections/analysis/Analysis.java

```diff
@@ -357,6 +357,30 @@ public double getSumDetailIntervalSize() {
     public double getSumDetailNumIntervals(){
         return intervalData.getNumIntervals();
     }
+	public double[][] getMsg_count() 
+    {
+	return intervalData.getMsg_count();
+    }
+	public double[][] getMsg_size() 
+    {
+	return intervalData.getMsg_size();
+    }
+	public double[][] getMsg_recv_count() 
+    {
+	return intervalData.getMsg_recv_count();
+    }
+	public double[][] getMsg_recv_size() 
+    {
+	return intervalData.getMsg_recv_size();
+    }
+	public double[][] getMsg_recv_count_ext() 
+    {
+	return intervalData.getMsg_recv_count_ext();
+    }
+	public double[][] getMsg_recv_size_ext() 
+    {
+	return intervalData.getMsg_recv_size_ext();
+    }
```

Comment:
Indentation and style, try to use camelCase and avoid using `_` in function/variable names if possible.